src/ansiblelint/config.py

```diff
@@ -60,6 +60,7 @@
     cwd=".",
     display_relative_path=True,
     exclude_paths=[],
+    format="rich",
```

Comment:
What does this have to do with removing prerun?

Reply:
It is needed, otherwise I will get a runtime errors when I try to access .format in main. In fact I am not sure how that it worked before this but I suspect it has something to do on config initialization order.